test/index.test.ts

```diff
@@ -1,92 +1,104 @@
+    probot.load(approvemanApp);
+  });
 
-  test('receive PR reopened', async (done) => {
-    setConfigNotFound()
-    checkApproved(done)
-    nock('https://api.github.com')
-      .get('/repos/tianhaoz95/approveman-test/pulls/1/files')
-      .reply(200, [
-        { filename: 'experimental/tianhaoz95/test.md' }
-      ])
-    await probot.receive({ name: 'pull_request', payload: prReopenedPayload })
-  })
+  test("receive PR reopened", async () => {
+    setConfigNotFound();
+    checkApproved();
+    checkSuccessStatus();
+    nock("https://api.github.com")
+      .get("/repos/tianhaoz95/approveman-test/pulls/1/files")
+      .reply(200, [{ filename: "experimental/tianhaoz95/test.md" }]);
+    await probot.receive({ name: "pull_request", payload: prReopenedPayload });
+  });
 
-  test('receive PR synchronize', async (done) => {
-    setConfigNotFound()
-    checkApproved(done)
-    nock('https://api.github.com')
-      .get('/repos/tianhaoz95/approveman-test/pulls/1/files')
-      .reply(200, [
-        { filename: 'experimental/tianhaoz95/test.md' }
-      ])
-    await probot.receive({ name: 'pull_request', payload: prSynchronizePayload })
-  })
+  test("receive PR synchronize", async () => {
+    setConfigNotFound();
+    checkApproved();
+    checkSuccessStatus();
+    nock("https://api.github.com")
+      .get("/repos/tianhaoz95/approveman-test/pulls/1/files")
+      .reply(200, [{ filename: "experimental/tianhaoz95/test.md" }]);
+    await probot.receive({
+      name: "pull_request",
+      payload: prSynchronizePayload,
+    });
+  });
 
-  test('receive PR opened', async (done) => {
-    setConfigNotFound()
-    checkApproved(done)
-    nock('https://api.github.com')
-      .get('/repos/tianhaoz95/approveman-test/pulls/1/files')
-      .reply(200, [
-        { filename: 'experimental/tianhaoz95/test.md' }
-      ])
-    await probot.receive({ name: 'pull_request', payload: prOpenedPayload })
-  })
+  test("receive PR opened", async () => {
+    setConfigNotFound();
+    checkApproved();
+    checkSuccessStatus();
+    nock("https://api.github.com")
+      .get("/repos/tianhaoz95/approveman-test/pulls/1/files")
+      .reply(200, [{ filename: "experimental/tianhaoz95/test.md" }]);
+    await probot.receive({ name: "pull_request", payload: prOpenedPayload });
+  });
 
-  test('read config', async (done) => {
-    setConfigToBasic('basic')
-    checkApproved(done)
-    nock('https://api.github.com')
-      .get('/repos/tianhaoz95/approveman-test/pulls/1/files')
-      .reply(200, [
-        { filename: 'docs/personal/tianhaoz95/test.md' }
-      ])
-    await probot.receive({ name: 'pull_request', payload: prOpenedPayload })
-  })
+  test("read config", async () => {
+    setConfigToBasic("basic");
+    checkApproved();
+    checkSuccessStatus();
+    nock("https://api.github.com")
+      .get("/repos/tianhaoz95/approveman-test/pulls/1/files")
+      .reply(200, [{ filename: "docs/personal/tianhaoz95/test.md" }]);
```

Comment:
Codacy found an issue: ['magic numbers' are not allowed: 200](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6083797)